lib/nimble_template/addons/github.ex

```diff
@@ -51,4 +51,44 @@ defmodule NimbleTemplate.Addons.Github do
 
     project
   end
+
+  @impl true
+  def do_apply(
+        %Project{
+          web_project?: web_project?,
+          mix_project?: mix_project?,
+          erlang_version: erlang_version,
+          elixir_version: elixir_version
+        } = project,
+        opts
+      )
+      when is_map_key(opts, :github_wiki) do
+    binding = [
+      web_project?: web_project?,
+      mix_project?: mix_project?,
+      erlang_version: erlang_version,
+      elixir_version: elixir_version
+    ]
+
+    template_getting_started_path =
+      if mix_project? do
+        ".github/wiki/Getting-Started.md.mix.eex"
+      else
+        ".github/wiki/Getting-Started.md.eex"
+      end
+
+    homepage_path = ".github/wiki/Home.md"
+    sidebar_path = ".github/wiki/_Sidebar.md"
+
+    files = [
+      {:text, ".github/workflows/publish_wiki.yml.eex", ".github/workflows/publish_wiki.yml"},
```

Comment:
This template file can be a normal file (without .eex), right? 🤔  

```suggestion
      workflow_path = ".github/workflows/publish_wiki.yml"

      {:text, workflow_path, workflow_path},
```

Reply:
Thanks for catching that, I've replaced the template file cfd0fdc28d2985265172c6eb01864e864ec80210 👍